packages/containers/psammead-timestamp-container/package.json

```diff
@@ -27,7 +27,9 @@
   "dependencies": {
     "@bbc/gel-foundations": "^3.4.0",
     "@bbc/psammead-timestamp": "^2.2.12",
-    "moment-timezone": "^0.5.26"
+    "@bbc/psammead-locales": "^2.23.0",
+    "moment-timezone": "^0.5.26",
+    "moment": "^2.24.0"
```

Comment:
This should be a peer dep I think :)

Reply:
With this, do you mean the `moment` one?

Reply:
Added just `moment` as a peerDep, `moment-timezone` needs to be a normal dependency.